src/main/java/Duke.java

```diff
@@ -1,10 +1,248 @@
+import Duke.exceptions.exceedNum;
+import Duke.tasks.Deadline;
+import Duke.tasks.Event;
+import Duke.tasks.Task;
+import Duke.tasks.Todo;
+
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+    public static void goodbye() {
+        System.out.println("\nBye.Hope to see you again soon!");
+    }
+
+    public static void greeting() {
+    /*String logo = " ____        _        \n"
```

Comment:
You can remove the unwanted code here to avoid cluttering.

Reply:
Thank you! I've removed it.